pkg/api/configsync/v1alpha1/conversion.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	conversion "k8s.io/apimachinery/pkg/conversion"
+	v1beta1 "kpt.dev/configsync/pkg/api/configsync/v1beta1"
+)
+
+// Convert_v1beta1_HelmRootSync_To_v1alpha1_HelmRootSync converts HelmRootSync
+// from v1beta1 to v1alpha1.
+//
+// This conversion is manual, because it is lossy.
+// The `DeployNamespace` field is in v1beta1, but not v1alpha1
```

Comment:
I wonder if this was intentional, I expect it may not have been. Maybe we should add the field to v1alpha1

Reply:
Seems fine to add new fields to new versions. The reason the automation makes you stub it out is to confirm that it's lossy. And in this case, the newer version is the hub/storage version, so nothing is lost when being stored, only if you request the older version.

Reply:
![You’re Killin Me Small Sandlot Porter](https://github.com/GoogleContainerTools/kpt-config-sync/assets/3980984/0a38c1ee-1532-4768-97c3-1da250a6416a)